test/over_react/util/cast_ui_factory_test.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@TestOn('browser')
+library cast_ui_factory_test;
+
+import 'package:over_react/over_react.dart';
+import 'package:test/test.dart';
+
+part 'cast_ui_factory_test.over_react.g.dart';
+
+main() {
+  group('castUiFactory', () {
```

Comment:
Should there be a test verifying that using `castUiFactory` to cast to a type that doesn't extend `UiFactory` would produce an error (assuming it would)?